src/LibVLCSharp.Forms/Shared/EqualizerUtils.cs

```diff
@@ -0,0 +1,192 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using LibVLCSharp.Shared;
+using Xamarin.Forms;
+
+namespace LibVLCSharp.Forms.Shared
+{
+    /// <summary>
+    /// Defines the <see cref="EqualizerUtils" />.
+    /// </summary>
+    public static class EqualizerUtils
```

Comment:
Can it be internal? As well as all its methods? We want to expose as little as possible generally.